ci/nightly/pipeline.template.yml

```diff
@@ -49,7 +49,8 @@ steps:
           composition: feature-benchmark
           args:
              - --other-tag
-             - latest
+             # common-ancestor will default to latest if not in a PR
```

Comment:
The `label` for this CI step needs to be reworded as now `latest` may or may not be the version that is being benchmarked against.

Reply:
✅ Renamed to: "Feature benchmark against merge base or 'latest'"